src/components/alert/alert.tsx

```diff
@@ -226,6 +226,10 @@ export class Alert implements OpenCloseComponent, LocalizedComponent, LoadableCo
             queued,
             [placement]: true
           }}
+          onMouseLeave={
```

Comment:
Can you use the pointer event equivalent instead of mouse event? `onPointerLeave/onPointerOver`?

I think `pointerout` might be better than pointerleave.